docs/specs/clients/core/identity/readme.md

```diff
@@ -0,0 +1,7 @@
+# Identity API
+
+## Description 
+
+Identity API exposes methods for managing [Identity Keys](identity-keys.md) on the keychain
+
+TODO: Add more context/description
```

Comment:
Anyone has an idea on how to describe it well?

Reply:
```suggestion
Identity API exposes methods for registering and un-registering [Identity Keys](identity-keys.md) on the keychain, that will be used by other protocols (Namely [chat](link to chat) and [push](link to push)). Those keys represent an identity of a blockchain account on a specific device.
```